app/storage/metadata_parser.py

```diff
@@ -81,22 +81,24 @@ def validate_metadata(claims, required_metadata):
     _validate_metadata_values_are_valid(claims, required_metadata)
 
 
-def _validate_metadata_is_present(metadata, required_metadata):
+def _validate_metadata_is_present(claims, required_metadata):
```

Comment:
I think for this PR it's probably worth leaving the names the same until we can make them all consistent as seeing claims and metadata in the same function is confusing